app.py

```diff
@@ -18,6 +18,41 @@
 class Main(Resource):
     def get(self):
         return "Nav-E API", 200
+
+@app.route('/login', methods = ['POST', 'GET'])
+def login():
+    if request.method == 'POST':
+        username = request.form.get('username')
+        password = request.form.get('password')
+        try: 
+            doc = db.collection('users').document(username).get()
+            if not doc.exists:
+                return "user does not exist", 404
+            if doc.to_dict()['password'] != password:
+                return "incorrect password", 401
+            return "login successful", 200
+        except:
+            return "failed to login"
+    
+    return make_response(render_template('login.html')), 200
```

Comment:
Lol api calls don't return html. Always a json

Reply:
You can't have ui with api calls. The ui is a separate app. The api will only return success or error and it's up to the ui service to render the login screen and stuff

Reply:
im gonna make a separate pull request for the auth stuff, but I moved the database stuff into a separate file. Where do i call the database initialization steps? Is that in app.py? I currently have anything related to databases in my new Trips.py file, but realized that I probably need it for the auth portion so I probably need to move it to app.py?

Reply:
nevermind i put it all in app.py and it worked